roles/sensu/server/templates/redis.json.j2

```diff
@@ -1,6 +1,6 @@
  {
    "redis": {
      "port": {{ redis_listen_port }},
-     "host": "{{ redis_interface }}"
+     "host": "{{ sensu_server_interface }}" 
```

Comment:
This changed due to the fact that the redis server can be listen to several interfaces, but on standalone configuration  sensu can only listen to one. Otherwise the HA with sentinel needs to be configured.